spec/features/catalog_spec.rb

```diff
@@ -586,17 +613,60 @@
     expect(page).to have_content('Restricted View')
   end
 
-  scenario 'local user should see the grey generic thumbnail and restricted access info' do
+  scenario 'local user should not see the grey generic thumbnail and still see restricted access info' do
```

Comment:
Could we change ` still see` to match it with the expected test result `expect(page).to_not have_content('Restricted View')`?

Reply:
@lsitu - I changed the expected test result but forgot to change the scenario.  I'll update it.  Thanks for catching it.

Reply:
@lsitu - I deployed the change with that suggested query to staging.  Other collections from #466 worked fine.  However, some mixed collection and some sample collections that Greg created didn't work.  For example - https://librarytest.ucsd.edu/dc/search?utf8=%E2%9C%93&q=visibility   As curator, I see access label = Restricted to UC San Diego use only, but if I click on that link I see access label = Some items restricted.  The old query also worked fast too.  I don't see any different in performance between the old and the new query.  Do you have any other suggestion that I can try?  Thanks